### ISSUE TITLE
Sound no longer works on Manjaro (kernel 6.6.x) and script  doesn't help

### ISSUE THREAD

**Issue by KRDucky** [WeirdTreeThing/chromebook-linux-audio#106]
**Describe the bug**

**Boardname**
If you don't know where to find this, run `cat /sys/class/dmi/id/product_name`

**Logs**
Generate logs with `wget https://raw.githubusercontent.com/chrultrabook/linux-tools/main/debugging.sh; bash debugging.sh` and attach the output here.

**Comment by KRDucky** [comment]
yup. I even tried rolling forward 1 kernel version and rolling back one. Neither kernel version helped.

**Comment by koluckirafal** [comment]
Hi @KRDucky, are you still experiencing this issue on newer system/kernel versions? If yes, can you provide logs for troubleshooting?

**Comment by KRDucky** [comment]
I don't know, I gave that laptop to a friend and haven't heard a complaint
since

**Comment by koluckirafal** [comment]
OK, thanks for answering :)
